### PR TITLE
Kelvinparmar patch 2

### PR DESCRIPTION
**Pull request by kelvinparmar**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2920"></a>

### CODE REVIEW

File: data/kelvinparmar.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/kelvinparmar",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://kelvin-parmar.hashnode.dev/",
+      "icon": "FaHashnode"
```
Comment (Aadarsh805) [inline data/kelvinparmar.json:26]: This Hashnode icon doesn't work use this instead `SiHashnode`

**Review by Njong392 — APPROVED** [review]
Looks great now! Thanks for contributing :tada: But there seems to be a merge conflict.

**Comment by kelvinparmar** [comment]
How can I resolve it? @Njong392

**Comment by eddiejaoude** [comment]
It looks like you didn't update your fork before making more changes, so you have a merge conflict with yourself.

Please read our docs on the best to update your profile, it has a step by step guide with screenshots
https://linkfree.eddiehub.io/docs/how-to-guides/editing

Your file already exists here https://github.com/EddieHubCommunity/LinkFree/blob/main/data/kelvinparmar.json

**Pull request closed without merging**